tomopy/prep/alignment.py

```diff
@@ -443,3 +455,240 @@ def shift_images(prj, sx, sy):
     prj *= scl
 
     return prj
+
+
+def find_slits_corners_aps_1id(img, 
+                               method='quadrant+', 
+                               medfilt2_kernel_size=3,
+                               medfilt_kernel_size=23,
+                               ):
+    """
+    Automatically locate the slit box location by its four corners.

```

Comment:
This needs more description here, because it seems this is very technique specific. Definition of the slits and its frame is needed. Also some explanation is needed about why you need corners of this box.

Reply:
The four slits that form a binding box is the current setup at aps_1id, which reduce the illuminated region on the detector.   Since the slits are stationary, they can serve as a reference to check detector drifting during the scan.  Technically, the four slits should be used to find the transformation matrix (not necessarily affine) to correct the image.  However, since we are dealing with 2D images with very little distortion, I use affine transformation matrices for approximation.  Therefore the "four corners" are used instead of all four slits.

Reply:
This is a nice explanation that we can use in the doc. :-)

Reply:
I add the comment into the docstring as a secondary description.